frontend/src/components/SubmissionHistory/SubmissionList.js

```diff
@@ -20,17 +19,19 @@ const SubmissionList = () => {
 
   const navigate = useNavigate();
 
-  const fetchData = async () => {
-    try {
-      const historyResponse = await getSubmissionHistory(await getCookie(), await getUserId());
-      setTableData(historyResponse.data.attemptsWithTitles);
-      setIsLoading(false);
-    } catch (error) {
-      errorHandler(error);
-    }
-  };
-
   useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const jwt = getCookie();
+        const userId = await getUserId();
+        const historyResponse = await getSubmissionHistory(jwt, userId);
```

Comment:
`getCookie()` is no longer awaited. Would this be a problem?

Reply:
getCookie dont require await